com.endlessnetwork.fablemaker.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id" : "com.endlessnetwork.fablemaker",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "18.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "build-options" : {
+    "strip" : true,
+    "no-debuginfo" : true
+  },
+  "command" : "com.endlessnetwork.fablemaker.sh",
+  "modules" : [
+    { 
+        "name" : "Fablemaker",
+        "buildsystem" : "simple",
+        "only-arches" : ["x86_64"],
+	"sources" : 
+	   [
+        {
+            "type": "git",
+            "url": "https://github.com/endless-network/Fablemaker_Binary.git",
+            "commit":"000a3bca06110c7f3dbd50305edadf8eea56fb22"
+        },
+        {
+            "type": "archive",
+            "url": "https://github.com/endless-network/Fablemaker_Binary/releases/download/v1.1.1/Fablemaker.zip",
+            "sha256":"519a4906ae585515de94376dd72e3ca7a5ca43fe34e12fd4d0ef77fe1762e3f7"
+        },
+        {
+            "dest-filename" : "com.endlessnetwork.fablemaker.sh",
+  	        "type" : "script",
+            "commands": 
+              [
+  	             "exec /app/bin/Fablemaker.x86_64"
+  	          ]
+
+  	      }
+     	],
+      "build-commands" : [
+        "install -Dm644 com.endlessnetwork.fablemaker.desktop /app/share/applications/com.endlessnetwork.fablemaker.desktop",
+        "install -Dm644 com.endlessnetwork.fablemaker.appdata.xml /app/share/appdata/com.endlessnetwork.fablemaker.appdata.xml",
+        "install -Dm644 FM_Favicon_128x128.png /app/share/icons/hicolor/128x128/apps/com.endlessnetwork.fablemaker.png",
+        "install -Dm644 FM_Favicon_64x64.png /app/share/icons/hicolor/64x64/apps/com.endlessnetwork.fablemaker.png",
+        "install -D com.endlessnetwork.fablemaker.sh /app/bin/com.endlessnetwork.fablemaker.sh",
+        "mv Fablemaker_Data /app/bin/",
+        "install Fablemaker.x86_64 /app/bin/Fablemaker.x86_64"
+        
+      ]
+    }
+      ],
+  "separate-locales" : false,
```

Comment:
we normally like these fields at the top of the file as it's a bit easier to reason about in review.